Sources/TuistSigning/SecurityController.swift

```diff
@@ -0,0 +1,63 @@
+import TSCBasic
+import TuistSupport
+
+protocol SecurityControlling {
+    func decodeFile(at path: AbsolutePath) throws -> String
+    func importCertificate(_ certificate: Certificate, keychainPath: AbsolutePath) throws
+    func createKeychain(at path: AbsolutePath, password: String) throws
+    func unlockKeychain(at path: AbsolutePath, password: String) throws
+    func lockKeychain(at path: AbsolutePath, password: String) throws
+}
+
+final class SecurityController: SecurityControlling {
```

Comment:
What about adding integration tests to this one? The test could do IO with a temporary keychain for testing purposes.